app/templates/components/paper-virtual-repeat.hbs

```diff
@@ -1,6 +1,6 @@
 {{#paper-virtual-repeat-scroller onScroll=(action 'onScroll')}}
 	<div class="md-virtual-repeat-sizer" style={{contentStyle}}></div>
 	<div class="md-virtual-repeat-offsetter" style={{concat "transform: " (if horizontal "translateX(" "translateY(")  _marginTop "px);"}}>
```

Comment:
This `style={{concat ...` triggers a warning for not using htmlSafe in style attribute.